apps/api/src/integration/plugins/slack.integration.ts

```diff
@@ -10,28 +12,73 @@ import {
 import { App } from '@slack/bolt'
 import { BaseIntegration } from './base.integration'
 import { PrismaService } from '@/prisma/prisma.service'
-import { decryptMetadata, makeTimedRequest } from '@/common/util'
+import { makeTimedRequest } from '@/common/util'
 import { InternalServerErrorException } from '@nestjs/common'
 
 export class SlackIntegration extends BaseIntegration {
-  private readonly app: App
+  private app: App
```

Comment:
I think we can get rid of this since this is not initialized anywhere. We could directly use the client in the functions